### PR TITLE
Use fixed from top-level mson element for JSON Schema

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

**Comment by pksunkara** [comment]
Fixes https://github.com/apiaryio/mson/issues/60

File: test/test-RenderTest.cc [hunk 1]
```diff
@@ -41,3 +41,6 @@ TEST_AST("render", "issue-312");
 
 TEST_AST("render", "issue-328-1");
 TEST_AST("render", "issue-328-2");
+
+TEST_AST("render", "fixed-attributes-section");
+TEST_AST("render", "fixed-named-type")
```
Comment (klokane) [inline test/test-RenderTest.cc]: no newline

**Comment by klokane** [comment]
Just formal problem, other things looks good

**Comment by pksunkara** [comment]
@klokane Updated.

**Comment by w-vi** [comment]
Shouldn't the tests be in schema section rather then render? Render is for JSON and schema for JSON Schema

**Comment by pksunkara** [comment]
Ah, I didn't realise we had a separate folder for schema which is weird because they can still be considered render.

**Comment by w-vi** [comment]
Well it is but it has different code tested so that's why it is separated, even though I think we should at some point improve the testing so that we need less fixtures covering more. Focused on the language constructs and not the results as we test many times the same blueprints in different places for different bits of the output.

**Pull request merged**